packages/icons/web/v2/boleto_24.svg

```diff
@@ -0,0 +1,28 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="24" height="24">
```

Comment:
This SVG is needlessly complex. I've simplified it:

```svg
<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
  <path
    d="M15.79 3.38308H16.316V20.3901H15.789L15.79 3.38308ZM14.79 3.38308H15.219V15.6021H14.789L14.79 3.38308ZM12.68 3.38308H13.113V15.6021H12.68V3.38308ZM13.57 3.38308H13.906V15.6021H13.57V3.38308ZM10.547 3.38308H10.98V15.6021H10.547V3.38308ZM7.676 3.38308H8.109V15.6021H7.676V3.38308ZM6.836 3.38308H7.051V15.6021H6.836V3.38308ZM4.777 3.38308H4.992V15.6021H4.777V3.38308ZM9.543 3.38308H9.762V15.6021H9.543V3.38308ZM5.71 3.38308H6.457V15.6021H5.711L5.71 3.38308ZM11.219 3.38308H11.961V15.6021H11.215V3.38208L11.219 3.38308ZM3.531 3.38308H3.965V15.6021H3.53L3.531 3.38308ZM2.379 3.38308H3.004V15.6021H2.379V3.38308ZM1.207 3.38308H1.641V15.6021H1.207V3.38308ZM0.0589981 3.38308H0.729998V15.6021H0.0599981L0.0589981 3.38308ZM8.488 3.38308H8.922V15.6021H8.488V3.38308ZM23.785 3.38308H24V20.3901H23.785V3.38308ZM21.723 3.38308H21.941V20.3901H21.723V3.38308ZM22.66 3.38308H23.398V20.3901H22.656V3.38308H22.66ZM20.48 3.38308H20.91V20.3901H20.48V3.38308ZM19.328 3.38308H19.953V20.3901H19.328V3.38308ZM18.156 3.38308H18.59V20.3901H18.156V3.38308ZM17.008 3.38308H17.68V20.3901H17.008V3.38308Z"
    fill="currentColor" />
</svg>
```

Reply:
hey @connor-baer 👋  the SVG you suggested for boleto does not contain the text, so I have changed it to the original version again

Reply:
Uh oh, good catch! It's weird, Figma just removes the text when importing the icon 🤔

The original version is overly complex though. Its use of ids can lead to issues on a page when the icon is displayed multiple times. I'll try optimizing it differently and commit it to your other PR (#1593).